data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -469,6 +469,35 @@ spec:
                           - high_performance
                           type: string
                       type: object
+                    powervs:
+                      description: PowerVS is the configuration used when installing
```

Comment:
Changes to this file will cause this test to fail. 
https://github.com/openshift/installer/blob/02ccd27538b080bcc09983f27dcb4aa42d26c5dc/pkg/explain/printer_test.go#L11

Reply:
not including this file caused the codegen checks to fail. i'll see if i can just include that one tweaked test and this file.

Reply:
Ah sorry I meant you should also update the test with these fields